doc/adr/0005-discovery-documents.md

```diff
@@ -0,0 +1,34 @@
+# 5. Use Discovery Documents for requirements and technical choices
+
+Date: 2019-10-11
+
+## Status
+
+Accepted
+
+## [Context](https://github.com/libero/community/issues/23)
+
+Libero product features and architectural choices benefit from the standardization of *discovery documents* that enable collaboration by being shared between team members, across teams, and outside of eLife.
+
+## Decisions
+
+- Always track discovery work in at least one dedicated Github issue.
+- Prefix Google Docs discovery documents titles with `Discovery document:`
+- Move them into the single [`Libero` shared drive](https://drive.google.com/drive/u/1/folders/0AC6f8fXdeXmMUk9PVA).
+- Link them from the relevant Github issue.
+- Establish a timebox in calendar time at the start of the discovery activity.
+- Organize in questions and answers; some of the answers are allowed to be assumption if clearly stated that they are.
```

Comment:
Think i'd prefer something like...
Capture questions and answers, and document any assumptions that have been made.